virtual_ecosystem/models/plants/plants_model.py

```diff
@@ -906,6 +1098,29 @@ def calculate_nutrient_uptake(self) -> None:
         self.data["plant_nitrate_uptake"] = self.data["dissolved_nitrate"] * 0.01
         self.data["plant_phosphorus_uptake"] = self.data["dissolved_phosphorus"] * 0.01
 
+        # Calculate N/P uptake (g N/P per stem) due to transpiration. Multiply:
+        # - Per stem transpiration (µmol H2O per stem)
+        # - Conversion factor from µmol H2O to m^3 (1.08015*10^-11)
```

Comment:
Technically speaking this varies with temperature and atmospheric pressure. Pyrealm supports the scaling, but I'm not sure we fix this in this PR. We do use this fine tuning elsewhere in the model but it is a pretty small scale effect.

Maybe raise a new issue?